exercises/pig-latin/Cargo.lock

```diff
@@ -0,0 +1,89 @@
+[root]
+name = "pig-latin"
+version = "0.1.0"
+dependencies = [
```

Comment:
related to the move to Cargo-example.toml, I would make the Cargo.lock based off of an empty `dependencies` in Cargo.toml so that the student does not see dependencies that they might not use